### DISCUSSION TITLE
Dataset Visibility Option Missing in 2.10.0a0? (Publishing workflow)

### DISCUSSION THREAD

**Discussion by disl-pdavid** [ckan/ckan#6921] [Q&A] [answered]
I'm working from master (e10534178) and noticed that the `visibility` option for the dataset is missing (was there in 2.9.5).

Is this intentional or just something waiting for someone to work on?

I'm also curious if that mechanism is the best approach for a workflow that requires review before being made public?
I'd like users to be able to create datasets but for the default visibility to be private until checked, then allowing an admin to publish publicly.

**Comment by amercader** [comment]
@disl-pdavid The Visibility field is there for me on the latest master. Are you using a custom schema with ckanext-scheming maybe? Or using custom auth settings?

**Comment by disl-pdavid** [comment]
No custom auth settings - stock CKAN running from master (7586d7868) using docker (`ckan 2.10.0a0`):

![image](https://user-images.githubusercontent.com/105015605/175357080-9a1f5e4b-30f5-4fcb-96db-b34d9ba00c7c.png)

Only extensions installed are xloader, geoview, spatial, and harvest.

**Comment by disl-pdavid** [comment, marked as answer]
Ah, I think I've found it - Visibility only shows up when the dataset is created as part of an organization.  Datasets not part of an org do not show this as an option (I think demo.ckan.org is the same way?).
